resources/definitions/julia.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "julia",
+    "name": "Julia",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Ultimaker",
+        "manufacturer": "Fracktal",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0]
+    },
+
+    "overrides": {
+        "machine_start_gcode": {
+            "default_value": " ;Basic settings: Layer height: {layer_height} Walls: {wall_thickness} Fill: {fill_density}\n ;Print time: {print_time}\n ;Filament used: {filament_amount}m {filament_weight}g\n ;Filament cost: {filament_cost}\n G21        ;metric values\n M107\n G28\n G29\n G90        ;absolute positioning\n G92 E0; reset extruder distance\n G1 Z5 F300 ;move nozzle up 5mm for safe homing\n G1 X0 Y0 Z0 F5000; move nozzle to home\n M300 S600P200\n M300 S800 P200\n M190 S{print_bed_temperature} ;Uncomment to add your own bed temperature line\n M109 S{print_temperature} ;Uncomment to add your own temperature line\n M82        ;set extruder to absolute mode\n M107       ;start with the fan off\n G1 Z15.0 F{travel_speed} ;move the platform down 15mm\n G92 E0                  ;zero the extruded length\n G1 F200 E3              ;extrude 3mm of feed stock\n G92 E0                  ;zero the extruded length again\n G1 F{travel_speed}\n ;Put printing message on LCD screen\n M117 Printing...\n"
```

Comment:
Some of these keys don't work any more, such as `fill_density` and `filament_amount`.